platform/ios/MGLAccountManager.m

```diff
@@ -51,8 +51,9 @@ + (void) setAccessToken:(NSString *) accessToken {
     [[MGLAccountManager sharedManager] setAccessToken:accessToken];
 
     // Update MGLMapboxEvents
-    // NOTE: This is (likely) the initial setup of MGLMapboxEvents
-    [MGLMapboxEvents sharedManager];
+    // NOTE: This is (likely) the initial setup of MGLMapboxEvents (via implicit call to +[MGLMapboxEvents sharedManager])
+    // Turn the Mapbox Turnstile to Count App Users
+    [MGLMapboxEvents pushEvent:MGLEventTypeAppUserTurnstile withAttributes:nil];
```

Comment:
`-setAccessToken:` can be called more than once. In particular, with #1553, if the developer decides to set the access token programmatically, `-setAccessToken:` will be called once with `nil` from within `+load` and again when the developer gets around to calling `+setAccessToken:`. You don’t want two turnstile events in that case.

A cleaner approach would be to leave the call to `+[MGLMapboxEvents sharedManager]` as is and just push a turnstile event at the end of `-[MGLMapboxEvents init]`.